dev-tools/scripts/smokeTestRelease.py

```diff
@@ -836,12 +827,15 @@ def checkIdenticalMavenArtifacts(distFiles, artifacts, version):
   for artifact in artifacts:
     if reJarWar.search(artifact):
       artifactFilename = os.path.basename(artifact)
+      if artifactFilename in ['solr-test-framework-%s.jar' % version]:
+        print('      Skipping artifact %s as it should not be in the binary distribution' % artifactFilename)
```

Comment:
do we want to explicitly test that it was not included? I thought there was a check against extra files but I can't find it now.

Reply:
Good idea - I added an explicit check (did not test it)